crates/sui-graphql-rpc/src/data/pg.rs

```diff
@@ -37,13 +44,20 @@ impl QueryExecutor for PgManager_ {
         U: Send + 'static,
     {
         let max_cost = self.limits.max_db_query_cost;
-        self.inner
+        let instant = Instant::now();
```

Comment:
When @amnn's refactor lands, would we want to track the time of the whole transaction, but should be a simple lift over to the execute and execute_repeatable methods

Reply:
Yes, there will be some changes once that lands. I am still playing with it but I think I got the gist.